lib/active_merchant/billing/gateway.rb

```diff
@@ -275,14 +275,14 @@ def localized_amount(money, currency)
         if non_fractional_currency?(currency)
           if self.money_format == :cents
             sprintf("%.0f", amount.to_f / 100)
-          else
+          elsif self.money_format == :dollars
```

Comment:
Curious about the explicit check for `:dollars`- since we only have `cents` and `dollars` in Active Merchant, then single `else` seems ok?